joomla/joomla-cms
# [6.2] Add webservices for guided tours

### Summary of Changes
Starts work on adding an API for guided tours to give a structured way to create/update/delete guided tours and their steps from the system

This currently roughly returns the right things. However, things left to do:

1. Check the step is part of the tour in the url when editing an individual step, else throw a 400.

### Testing Instructions
Install copy of joomla. Check API endpoints - these will all be covered by the tests once completed

### Actual result BEFORE applying this Pull Request
No API exists

### Expected result AFTER applying this Pull Request
API exists

### Link to documentations
Please select:

- [x] Pull Request link for manual.joomla.org: TODO!
